pkgdb/Makefile

```diff
@@ -145,7 +145,7 @@ endif  # ifndef TOOLCHAIN
 # Read info from files
 # --------------------
 
-VERSION := $(file < $(PKGDB_ROOT)/version)
```

Comment:
This avoids an issue where `#include <version>` inside of `boost` can wind up including this file if you are using a non-GNU version of `make`.